.github/workflows/jest-testing.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Jest testing suite
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+
+env:
+  NODE_ENV: 'test'
+  TWITTER_API_KEY: '<twitter_api_key>'
```

Comment:
Why do we need to pass twitter API keys?

Reply:
In the twitter helper, the file start as such
```TypeScript
if (!apiKey || !apiKeySecret || !accessToken || !accessTokenSecret) {
  throw new Error("Twitter environment variables are not set");
}
```
so if the values are not there every test would fail, so I thought it would be more convenient to have them defined.